concrete/controllers/single_page/login.php

```diff
@@ -388,6 +391,20 @@ public function logout($token = false)
         }
     }
 
+    /**
+     * @param $token
+     * @return RedirectResponse
+     */
+    public function do_logout($token = false)
```

Comment:
Why do we need a whole new method for this? Why can't we use the old one? the behavior is the same... right?